### DISCUSSION TITLE
2FA remember for certain amount of time

### DISCUSSION THREAD

**Discussion by lzinga** [requarks/wiki#6459] [Help / Questions] [answered]
I have 2FA enabled and it asks me every time I want to login. Is there any way to enable a duration so on trusted computers I don't have to type in the code every time? It makes a quick look up for a piece of information a lot longer.

**Comment by NGPixel** [comment]
You shouldn't need to login as long as you're using the wiki at least once in the long expiration period (default 2 weeks)...

**Reply by lzinga** [reply]
Hmmm, then something must be causing it to ask every time based on how I am hosting it. I will look into it a bit and see if I notice anything.

**Reply by lzinga** [reply, marked as answer]
well... This is embarrassing. After looking at all the pieces apparently it was the fact I bookmarked the login page itself, not the root url. Although it probably is rare, would it be beneficial to add to the login page to check if the user is already logged in or has token information and then redirect to the wiki itself?

**Reply by NGPixel** [reply]
No, as there're cases where you want to login as another user.
